ports-wip/openssh/build.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# This code Copyright 2012 Todd Mortimer <todd.mortimer@gmail.com>
+#
+# You may do whatever you like with this code, provided the above
+# copyright notice and this paragraph are preserved.
+
+
+set -e
+source ../../lib.sh
+#DISTVER="openssh-7.1p2"
+#DISTVER="openssh-7.3p1"
+DISTVER="openssh-7.9p1"
```

Comment:
Maybe rename the `ports-wip/openssh` directory to `ports-wip/openssh-update` to avoid confusion?

Reply:
done (local branch)